build.psm1

```diff
@@ -924,11 +922,11 @@ function Install-Dotnet {
             Write-Warning "This script only removes prior versions of dotnet for Ubuntu 14.04 and OS X"
         }
 
-        # Install new dotnet 1.0.0 preview packages
+        # Install new dotnet 1.1.0 preview packages
         $installScript = "dotnet-install.sh"
         Start-NativeExecution {
             curl -sO $obtainUrl/$installScript
-            bash ./$installScript -c $Channel -v $Version
+            bash ./$installScript -c $Channel
```

Comment:
I think I had removed this whilst trying to get the 1.1 install script working, but the defaults are now set correctly elsewhere in build.psm1, so it can be restored.